--- index.php.orig	2012-09-20 15:51:02.508680629 +0200
+++ index.php	2012-09-06 13:45:12.555544956 +0200
@@ -86,6 +86,10 @@
 		trigger_error("CakePHP core could not be found.  Check the value of CAKE_CORE_INCLUDE_PATH in APP/webroot/index.php.  It should point to the directory containing your " . DS . "cake core directory and your " . DS . "vendors root directory.", E_USER_ERROR);
 	}
 
+	if (isset($_SERVER['PATH_INFO']) && $_SERVER['PATH_INFO'] == '/favicon.ico') {
+		return;
+	}
+
 	App::uses('Dispatcher', 'Routing');
 
 	$Dispatcher = new Dispatcher();
