--- app/Config/database.php.default	2012-09-01 10:28:02.000000000 +0200
+++ app/Config/database.php	2012-09-25 12:52:54.704269036 +0200
@@ -62,21 +62,11 @@
 	public $default = array(
 		'datasource' => 'Database/Mysql',
 		'persistent' => false,
-		'host' => 'localhost',
-		'login' => 'user',
-		'password' => 'password',
-		'database' => 'database_name',
-		'prefix' => '',
-		//'encoding' => 'utf8',
-	);
-
-	public $test = array(
-		'datasource' => 'Database/Mysql',
-		'persistent' => false,
-		'host' => 'localhost',
-		'login' => 'user',
-		'password' => 'password',
-		'database' => 'test_database_name',
+		'host' => '127.0.0.1',
+		'login' => 'cyberdefence',
+		'port' => 8889,
+		'password' => 'PhwaxFJbNhnRGfZ8',
+		'database' => 'cydefsig',
 		'prefix' => '',
 		//'encoding' => 'utf8',
 	);
