--- app/View/Layouts/Emails/text/default.ctp.orig	2012-09-25 14:09:01.470702330 +0200
+++ app/View/Layouts/Emails/text/default.ctp	2012-09-25 14:09:22.918702841 +0200
@@ -16,6 +16,4 @@
  * @license       MIT License (http://www.opensource.org/licenses/mit-license.php)
  */
 ?>
-<?php echo $this->fetch('content');?>
-
-This email was sent using the CakePHP Framework, http://cakephp.org.
+<?php echo $this->fetch('content');?>
\ No newline at end of file
