--- app/Config/core.php.orig	2012-05-22 18:37:44.000000000 +0200
+++ app/Config/core.php	2012-09-21 08:23:07.000000000 +0200
@@ -110,13 +110,13 @@
  *	`manager_index()` and `/manager/controller/index`
  *
  */
-	//Configure::write('Routing.prefixes', array('admin'));
+    Configure::write('Routing.prefixes', array('admin'));
 
 /**
  * Turn off all caching application-wide.
  *
  */
-	//Configure::write('Cache.disable', true);
+	Configure::write('Cache.disable', false);
 
 /**
  * Enable cache checking.
@@ -184,12 +184,12 @@
 /**
  * A random string used in security hashing methods.
  */
-	Configure::write('Security.salt', 'DYhG93b0qyJfIxfs2guVoUubWwvniR2G0FgaC9mi');
+	Configure::write('Security.salt', 'Rooraenietu8Eeyo<Qu2eeNfterd-dd+');
 
 /**
  * A random numeric string (digits only) used to encrypt/decrypt strings.
  */
-	Configure::write('Security.cipherSeed', '76859309657453542496749683645');
+	Configure::write('Security.cipherSeed', '395786739573056621429506834955');
 
 /**
  * Apply timestamps with the last modified time to static assets (js, css, images).
  