--- app/Config/routes.php.orig	2012-05-22 18:37:44.000000000 +0200
+++ app/Config/routes.php	2012-09-25 12:59:05.832275906 +0200
@@ -25,14 +25,17 @@
  * its action called 'display', and we pass a param to select the view file
  * to use (in this case, /app/View/Pages/home.ctp)...
  */
-	Router::connect('/', array('controller' => 'pages', 'action' => 'display', 'home'));
-/**
- * ...and connect the rest of 'Pages' controller's urls.
- */
-	Router::connect('/pages/*', array('controller' => 'pages', 'action' => 'display'));
+	Router::connect('/', array('controller' => 'events', 'action' => 'index'));
+
+	// admin pagination
+	Router::connect('/users/admin_index/*', array('controller' => 'users', 'action' => 'index', 'admin' => true));
+
+	// Activate REST
+	Router::mapResources(array('events'));
+	Router::parseExtensions('xml');
 
 /**
- * Load all plugin routes.  See the CakePlugin documentation on 
+ * Load all plugin routes.  See the CakePlugin documentation on
  * how to customize the loading of plugin routes.
  */
 	CakePlugin::routes();
