--- lib/Cake/View/MediaView.php.orig	2012-12-12 16:45:24.269525098 +0100
+++ lib/Cake/View/MediaView.php	2012-12-12 16:46:32.173528779 +0100
@@ -78,11 +78,13 @@
 		$name = $download = $extension = $id = $modified = $path = $cache = $mimeType = $compress = null;
 		extract($this->viewVars, EXTR_OVERWRITE);
 
-		if (is_dir($path)) {
+		// deresz: commented out for compatibility with NCIRC PHP security settings (open_basedir restriction)
+
+		//if (is_dir($path)) {
 			$path = $path . $id;
-		} else {
-			$path = APP . $path . $id;
-		}
+		//} else {
+		//	$path = APP . $path . $id;
+		//}
 
 		if (!is_file($path)) {
 			if (Configure::read('debug')) {
